flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java

```diff
@@ -100,7 +100,9 @@
     /** Name for queries against state created from this StateDescriptor. */
     @Nullable private String queryableStateName;
 
-    /** Name for queries against state created from this StateDescriptor. */
+    /**
+     * The state time-to-live(TTL). If the setting is not displayed, the ttl will not be turned no.
```

Comment:
```suggestion
     * The configuration of state time-to-live(TTL), it is disabled by default.
```

Reply:
Thank you for you recovery. I also think you suggestion is more concise.